catalog/templates/column.html

```diff
@@ -27,112 +27,232 @@
+
+      Obtain an estimate of "{{ col.name }}" per square kilometer using area
+      weighted interpolation.
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_Buffer(ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326), 0.01) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}",
+               "normalization": "areaNormalized"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as per_sq_km
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
+
+   .. tab:: {"tab_name": "Raw count"}
+
+      Obtain an estimate of "{{ col.name }}" for a polygon using area-weighted
+      interpolation.
+
+      .. code-block:: postgresql
+
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_Buffer(ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326), 0.01) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}",
+               "normalization": "predenominated"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as per_sq_km
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
+
+
+{% else %}
+   .. tab:: {"tab_name": "Point"}
+
+      Obtain the value of "{{ col.name }}" for a point.  The value will be from
+      the densest bounday with this measure.
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as val
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
 
-.. code-block:: postgresql
+{% endif %}
 
-  INSERT INTO {new_table_name} (the_geom, {geo_id_column})
-  SELECT *
-  FROM OBS_GetBoundariesByGeometry(
-    ST_Buffer({{ point }}, 0.01),
-    '{{ col.id }}'
-  )
+{#
+   .. tab:: {"tab_name": "Boundaries within polygon "}
 
-{% endif %}
+      Obtain values of "{{ col.name }}" for boundaries within a polygon.
 
-{% for denom_reltype, denom_id, denom_name in col.denoms %}
-{% if denom_name and denom_id %}
-Measure "{{ col.name }}" percent of "{{ denom_name }}" at one point:
+      .. code-block:: postgresql
 
-.. code-block:: postgresql
+          TODO
```

Comment:
It is commented (`{#`) and the equivalent code is [above](https://github.com/CartoDB/bigmetadata/pull/164/files#diff-d70579600c06a04ff503924c40445bb4R34). I'll remove it. 🔥